setup/server.py

```diff
@@ -68,7 +68,8 @@ def wizard():
         @bp.route("/submit_flavor", methods=["POST"])
         def submit_flavor():
             data = flask.request.form.copy()
-            return flask.render_template('wizard.html', flavor=data["flavor"])
+            flavor_files = sorted(os.listdir(path + "/" + version + "/templates/steps/" + data["flavor"]))
+            return flask.render_template('wizard.html', flavor=data["flavor"], flavor_files=flavor_files)
```

Comment:
@kaiyou, we are trying to pass two variables down to the template here.The `flavor` and a list containing the steps for that flavor: `flavor_files`. Is there a more elegant way we could do this?

Reply:
@ionutfilip, lets rename `flavor_files` to `steps`.

Reply:
I do not believe there is a more elegant way, it is already quite acceptable :)